recipes/shenfun/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set build = 0 %}
+{% set name = "shenfun" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spectralDNS/shenfun/archive/{{ version }}.tar.gz
+  sha256: 48e51882b2d45359997aee4de3d7dc46c81db2ee46b5ddfb330ebdb81ba312ad
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - numpy 1.16.*
+    - setuptools
+    - python
+    - cython
+    - mpi4py
+    - mpi4py-fft
+    - {{ mpi }}
+    - fftw
+    - pip
+    - scipy
```

Comment:
Are all this really needed to build? Are you linking against SciPy?

Reply:
Hi @scopatz, thanks for looking at this. 
The thing is that I have scipy as `install_requires` in the [setup](https://github.com/spectralDNS/shenfun/blob/426206bae35b5707c0946bb60244656f0735ee72/setup.py#L99), and as long as it's there it also has to be in the meta. I could probably build without it though. But it is used heavily and it made sense at the time to make it a requirement.

Reply:
right, but you have `pip install --no-deps` above, so what is in the `install_requires` should be ignored for the build/host phase.  It is generally a good idea to keep build and host as minimal as possible. It just seems like there are a lot of extra packages here that you don't really need, even if they should be in the run requirements.  If you are willing to play around and minimize this list that'd be great! If not, let me know.

Reply:
I'll see what I can do.

Reply:
Excellent! Thanks!